cogip/tools/monitor/robots.py

```diff
@@ -32,11 +32,6 @@ def __init__(self, game_view: GameView):
         self._round_obstacles_pool: List[DynCircleObstacleEntity] = []
         self._lidar_emulation: Dict[int, bool] = {}
 
-        # Provision 2 robots
-        for i in range(2):
-            self.add_robot(i + 1)
-            self.del_robot(i + 1)
-
```

Comment:
should not be removed in this commit

Reply:
Yes it should ;-)
Provisioning robot entities at this stage prevent robot detection from starting, it is too early.